.env.testnet

```diff
@@ -75,10 +75,10 @@ L2_ETHERSCAN_API_KEY=
 # *************** PRIVATE TESTNET ***************
 
 # RPC URL of the L1 network from which a private L1 test network is forked if a private test network is used, e.g. Infura, Alchemy
-L1_FORK_RPC_URL=https://sepolia.infura.io/v3/YOUR_INFURA_KEY
+L1_FORK_RPC_URL=https://ethereum-sepolia-rpc.publicnode.com
 
 # RPC URL of the L2 network from which a private L2 test network is forked if a private test network is used, e.g. Infura, Alchemy
-L2_FORK_RPC_URL=https://optimism-goerli.infura.io/v3/YOUR_INFURA_KEY
+L2_FORK_RPC_URL=https://rpc.sepolia-api.lisk.com
```

Comment:
Same as above: I'm not sure `.env.testnet` should be used to setup a private testnet.

Reply:
This is not used for private testnets but for public testnets (Sepolia specifically)

Reply:
Ah sorry, I see your point now. Makes sense, I will remove it